opensource/project/software-req-win.md

```diff
@@ -65,6 +97,43 @@ To test it, stop the system Docker daemon and start the one you just built:
 
 The other make targets work too, to run unit tests try: `docker run --rm docker-builder sh -c 'cd /c/go/src/github.com/moby/moby; hack/make.sh test-unit'`.
 
+### 6. (Optional) Remove the interim container holding the built executable binaries:
+
+    docker rm binaries
+    
+### 7. (Optional) Remove the image used for the container in which the executable binaries are build. 
+
+Tip - it may be useful to keep this image around if you need to build multiple times. Then you can take advantage of the builder cache to have an image which has all the components required to build the binaries already installed.
+
+    docker rmi nativebuildimage    
+    
+## Validation
+
+The validation tests can only run directly on the host. This is because they calculate information from the git repo, but the .git directory is not passed into the image as it is excluded via .dockerignore. Run the following from a Windows PowerShell prompt (elevation is not required): (Note Go must be installed to run these tests)
```

Comment:
Please wrap lines at or near 80 characters so that when this content is reviewed in the future, the review can be more granular.